modules/calib3d/test/test_affine2d_estimator.cpp

```diff
@@ -127,4 +127,32 @@ TEST_P(EstimateAffine2D, testNPoints)
     }
 }
 
+// test conversion from other datatypes than float
+TEST_P(EstimateAffine2D, testConversion)
+{
+    Mat aff(2, 3, CV_32S);
+    cv::randu(aff, 1., 3.);
```

Comment:
Please, change `randu` to
```
RNG rng(111);
rng.fill(aff, RNG::UNIFORM, 1., 3.);
```
`randu` uses global RNG and it's state depends on previous calls.